test/e2e/launch.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+if [ -z "$TAG" ]
+then
+TAG=latest
+fi
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+DIR="$(cd "$(dirname "${0}")" && pwd)"
+CURRENT=$PWD
+cd "$DIR/../.."
+
+# Load the image
+make build
```

Comment:
the .travis.yml already run: `make build` and `make container` so maybe you can add an option to this script in order to skip this two target in case of travis-ci run.